src/jdk.internal.vm.ci/share/classes/jdk/vm/ci/code/RegisterConfig.java

```diff
@@ -94,6 +94,13 @@ default int getMaximumFrameSize() {
      */
     RegisterArray getCalleeSaveRegisters();
 
+    /**
+     * Gets the storage kind for a callee save register.
```

Comment:
I would add a second sentence describing the Window ABI example so that it's clear why this API exists.